components/http_foundation.rst

```diff
@@ -377,6 +377,14 @@ method takes an instance of
 You can clear a cookie via the
 :method:`Symfony\\Component\\HttpFoundation\\ResponseHeaderBag::clearCookie` method.
 
+You can also create a 
+:class:`Symfony\\Component\\HttpFoundation\\Cookie` object from a raw header value using :method:`Symfony\Component\HttpFoundation\Cookie::fromString`.
+
+.. tip::
```

Comment:
I still think we should not include this because:

* We don't do the same in other cases. We always use the OOP way of dealing with request/response, instead of going deep below to deal with HTTP headers directly.
* We must always be careful with all the examples we include because if we show *a way* to do things, some people will think that's *the way* to do things.

@xabbuh what's your opinion about this? Thanks!